osquery/tables/system/linux/cpu_time.cpp

```diff
@@ -0,0 +1,86 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+#include <vector>
+#include <boost/filesystem.hpp>
```

Comment:
I don't think you need these boost includes. The `split` method comes from osquery's conversions. `boost::trim` and `::starts_with` / `::is_any_of` should be inherited from the conversions too.